src/PlaywrightSharp/Helpers/TaskUtils.cs

```diff
@@ -1,35 +1,35 @@
-using System.Threading.Tasks;
-
-namespace PlaywrightSharp.Tests
-{
-    internal static class TaskUtils
-    {
-        public static async Task<(T1, T2)> WhenAll<T1, T2>(Task<T1> task1, Task<T2> task2)
-        {
-            await Task.WhenAll(task1, task2).ConfigureAwait(false);
-
-            return (task1.Result, task2.Result);
-        }
-
-        public static async Task<T> WhenAll<T>(Task<T> task1, Task task2)
-        {
-            await Task.WhenAll(task1, task2).ConfigureAwait(false);
-
-            return task1.Result;
-        }
-
-        public static async Task<(T1, T2, T3)> WhenAll<T1, T2, T3>(Task<T1> task1, Task<T2> task2, Task<T3> task3)
-        {
-            await Task.WhenAll(task1, task2, task3).ConfigureAwait(false);
-
-            return (task1.Result, task2.Result, task3.Result);
-        }
-
-        public static async Task<(T1, T2)> WhenAll<T1, T2>(Task<T1> task1, Task<T2> task2, Task task3)
-        {
-            await Task.WhenAll(task1, task2, task3).ConfigureAwait(false);
-
-            return (task1.Result, task2.Result);
-        }
-    }
-}
+using System.Threading.Tasks;
+
+namespace PlaywrightSharp.Helpers
+{
+    internal static class TaskUtils
```

Comment:
I want to keep tasks on the libraries close to the metal. I would use this only for tests.

Reply:
see how it's used in `ElementHandle.ViewportPointAndScrollAsync` I don't really see a reason not to include this as an internal helper class

Reply:
Tracking task issues can be tricky and adding one more layer can make it even more complicated. I want to be as close as the API as possible. We can solve `ViewportPointAndScrollAsync` without a Task utils.